src/js/popup/bind/remove-button.js

```diff
@@ -0,0 +1,35 @@
+import { decreaseAvailableCount } from 'popup/render/available-count';
+
+const bindRemoveButton = messageClient => {
```

Comment:
The logic of the remove button is clearly easier to understand. 
Nice refactoring!!😭